balancer/pickfirst/pickfirstleaf/pickfirstleaf_ext_test.go

```diff
@@ -851,6 +851,114 @@ func (s) TestPickFirstLeaf_EmptyAddressList(t *testing.T) {
 	}
 }
 
+func (s) TestPickFirstLeaf_InterleavingIPV4Preffered(t *testing.T) {
```

Comment:
A test for unknown family being preferred?

Reply:
Added a test.